activities/supporting-codebase-governance/customized-governance-workshop/index.md

```diff
@@ -0,0 +1,46 @@
+---
+type: Index
+---
+
+# Customized governance workshop
+
+A customized governance workshop can be used for a codebase that already have some governance in place.
+It may be a draft or already in use, work fine or have some problems.
+The state of the governance may suggest what to focus the workshop on but it's strength is leveraging the governance game to socialize understanding of the governance.
+It can be used to highlight conflicting views by stress testing the governance which may make it easier to develop a path forward.
```

Comment:
```suggestion
As such, it can be used to simply sketch out a currently well functioning mechanism, talk through an initial draft or proposed changes, or used to surface tensions, pain points, or differing understandings. 
It may also be used to talk through different scenarios to stress test the governance mechanism.
```